includes/discovery/sensors/pre-cache/ipoman.inc.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+$pre_cache['ipoman']['out'] = $pre_cache['ipoman']['out'] ?? [];
+$pre_cache['ipoman']['in'] = $pre_cache['ipoman']['in'] ?? [];
 echo 'outletConfigDesc ';
 $pre_cache['ipoman']['out'] = snmpwalk_cache_multi_oid($device, 'outletConfigDesc', $pre_cache['ipoman']['out'], 'IPOMANII-MIB');
```

Comment:
```suggestion
$pre_cache['ipoman']['out'] = snmpwalk_cache_multi_oid($device, 'outletConfigDesc', [], 'IPOMANII-MIB');
```

Reply:
This is easier to fix,   just copy paste people not realizing the argument is an array to merge into.  Just start with an empty array on the first calls instead.